pkg/registry/scripts/images.js

```diff
@@ -36,8 +38,15 @@
 
     .controller('ImagesCtrl', [
         '$scope',
-        function($scope) {
+        'kubeLoader',
+        'kubeSelect',
+        function($scope, loader, select) {
+
             /* nothing here yet */
+            loader.watch("images");
```

Comment:
I don't think that's how ```loader.watch()``` accepts its arguments. See the documentation in comments near kubeLoader.